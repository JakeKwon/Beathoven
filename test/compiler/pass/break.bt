int x = 5;

while (x < 10) {
  if (x == 8){
    print("should break");
    break;
  }
  print(x, "\n");
  x = x + 1;

}