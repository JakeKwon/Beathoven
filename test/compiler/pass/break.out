5
6
7
should break